specification/storage/data-plane/Microsoft.StorageDataLake/stable/2019-12-12/DataLakeStorage.json

```diff
@@ -2189,7 +2189,29 @@
+                "type": "string"
+              }
+            }
+          },
+          "202": {
+            "description": "Delete request is accepted, applicable only when Hierarchical Namespace is enabled for the account.",
+            "headers": {
+              "Date": {
+                "description": "A UTC date/time value generated by the service that indicates the time at which the response was initiated.",
+                "type": "string"
+              },
+              "x-ms-request-id": {
+                "description": "A server-generated UUID recorded in the analytics logs for troubleshooting and correlation.",
+                "pattern": "^[{(]?[0-9a-f]{8}[-]?([0-9a-f]{4}[-]?){3}[0-9a-f]{12}[)}]?$",
+                "type": "string"
+              },
+              "x-ms-version": {
+                "description": "The version of the REST protocol used to process the request.",
+                "type": "string"
+              },
+              "x-ms-continuation": {
+                "description": "Applicable only when Hierarchical Namespace is enabled for the account. When deleting a directory, the number of paths that are deleted with each invocation is limited.  If the number of paths to be deleted exceeds this limit, a continuation token is returned in this response when the following conditions are met: Delete should be on a directory with query parameters \"recursive\" and \"paginated\" set to true, the identity of the user calling the api should be a non-super user using ACL based authorization, number of paths to be deleted exceeds a limit, request version is 2023-08-03 and later. When a continuation token is returned in the response, it must be specified in a subsequent invocation of the delete operation to continue deleting the directory. Directory is successfully deleted when the continuation token returned is empty. The actual directory deletion happens only in the last invocation, the previous ones involve ACL checks in the server of the files and directories under the directory to be recursively deleted.",
```

Comment:
The actual directory deletion happens only in the last invocation, the previous ones involve ACL checks in the server of the files and directories under the directory to be recursively deleted.

Same comment as above